api/ngrok/v1alpha1/kubernetesoperator_types.go

```diff
@@ -115,6 +137,8 @@ type KubernetesOperatorSpec struct {
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:name="ID",type=string,JSONPath=`.status.id`,description="Kubernetes Operator ID"
+// +kubebuilder:printcolumn:name="Name",type="string",JSONPath=".metadata.name"
```

Comment:
We can remove `Name` here. It is already in the output by default and adding it causes it to be printed in the output twice.